interfaces/Portalicious/src/app/components/form/form-sidebar.component.html

```diff
@@ -1,34 +1,37 @@
 <p-sidebar
   [(visible)]="visible"
   position="right"
```

Comment:
I can see a built-in fix for it. I guess the fix on our end could be to conditionally make it "left" or "right" based on RTL mode. But tbh I would leave these kind of ad-hoc rtl fixes for if/when we have to support an RTL language - then we can check again what the status of primeng is on this because currently the support is not great and they have quite an inconsistent way of dealing with it.